.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Is there a reason to include the devcontaine.json file with the project in GitHub? or is this something that accidentally made it into the branch.

Reply:
Now that we have dockerfile, deleted this file from the repo.